.github/workflows/ci.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           fetch-depth: 0
 
       - name: Set up Python
-        uses: actions/setup-python@v2.2.2
+        uses: actions/setup-python@v2
```

Comment:
I don't really understand why this needs to change? I saw your comment on #533 - but it doesn't make sense to me... Dependabot said "Hey, update this to 2.2.2" and I said "Yeah, cool, makes sense, doesn't seem to break anything", and there we were. 🤷

What is the benefit to dropping it back down to `v2`?

Reply:
Dependabot is stupid in this case. It doesn't take into account that there's a `v2` tag and it will update to a point release wrongfully.

This makes sure we always get the latest v2 version without having to bother with each point release.

Reply:
Right. So `@v2` includes everything that `@v2.2.2` does?

Reply:
Answered my own question. Cool. Good to know for future. (Who am I kidding, I won't remember...)

![image](https://user-images.githubusercontent.com/1998970/125171121-64659180-e1aa-11eb-8ac7-405f7b2cff84.png)

Reply:
No worries, it's a bug I have reported months ago and they are supposed to be working on it, but priorities I guess.